batavia/types/Slice.js

```diff
@@ -52,6 +54,18 @@ Slice.prototype.__str__ = function() {
 }
 
 /**************************************************
+ * Operands
+ **************************************************/
+
+Slice.prototype.__add__ = function(other) {
+    var types = require('../types')
+    
+    throw new exceptions.TypeError.$pyclass(
+        'unsupported operand type(s) for +: \'slice\' and \'' + type_name(other) + '\''
+    )
+}
+
```

Comment:
At column 1: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)